packages/paste-core/components/button/src/index.tsx

```diff
@@ -69,34 +89,40 @@ const Button: React.FC<ButtonProps> = props => {
     }
   }
 
-  handlePropValidation(props);
+  handlePropValidation({as, href, variant, size, tabIndex, fullWidth, children});
```

Comment:
Same here, comma separated list of arguments and modify the function

Reply:
Same. Restructured. Let me know if I have that setup correctly.